application/services/sms_service.py

```diff
@@ -7,53 +7,44 @@
 import smtplib
 import os
 
-def send_message(camera, timestamp, feed_url, img_filename):
+def send_detection_message(camera, timestamp):
     """
-    Sends an SMS message via a carrier through SMTP. 
+    Sends an SMS message via a carrier through SMTP for detection.
     """
-    text = svc_common.get_detection_message(camera, timestamp, feed_url);
+    text = svc_common.get_detection_message(camera, timestamp);
+    send_message(text)
 
+def send_message(text, img_filename=None, recipients=None):
     msg = MIMEMultipart()
+    msg['Subject'] = "/"
```

Comment:
Subject needs to be set to something, otherwise a "no subject" string shows up in the SMS